src/Conference/ConferenceDetails.jsx

```diff
@@ -73,6 +75,18 @@ const ConferenceDetails = () => {
   const { name, topic, dateTime, city, description } = conference || {};
   const date = new Date(dateTime);
 
+  const deleteAConference = async id => {
+    try {
+      await deleteConference(id);
+      history.push("/");
+    } catch (error) {
+      console.log(error);
+    }
+  };
+  const onClick = () => {
```

Comment:
I probably would be more declerative with what its actually supposed to do, perhaps `deleteConference` so you can see when glancing at the button what it actually does